src/sentry/models/groupsnooze.py

```diff
@@ -142,6 +139,42 @@ def test_user_rates(self):
 
         return True
 
+    def test_user_counts(self, group=None):
+        if features.has(
+            "organizations:groupsnooze-cached-counts", organization=self.group.project.organization
+        ):
+            return self.test_user_counts_w_cache(group)
+        else:
+            return self.test_user_counts_no_cache(group)
+
+    def test_user_counts_no_cache(self, group=None):
+        metrics.incr("groupsnooze.test_user_counts")
+        metrics.incr("groupsnooze.test_user_counts.real_count")
+
+        threshold = self.user_count + self.state["users_seen"]
+        real_count = group.count_users_seen(
+            referrer=Referrer.TAGSTORE_GET_GROUPS_USER_COUNTS_GROUP_SNOOZE.value
+        )
+        return real_count <= threshold
+
+    def test_user_counts_w_cache(self, group=None):
```

Comment:
Is the plan to rename this after we remove the flag?

Reply:
yeah, when we remove the flag the shim goes away and this one is renamed `test_user_counts`